server/sonar-server/src/main/java/org/sonar/server/test/index/TestIndex.java

```diff
@@ -52,7 +53,7 @@ public TestIndex(EsClient client) {
       .setSize(1)
       .setQuery(boolQuery().must(matchAllQuery()).filter(termQuery(FIELD_TEST_UUID, testUuid)))
       .get().getHits().getHits()) {
-      coveredFiles.addAll(new TestDoc(hit.sourceAsMap()).coveredFiles());
+      coveredFiles.addAll(new TestDoc(hit.getSourceAsMap()).coveredFiles());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "sourceAsMap"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)